webrtc.html

```diff
@@ -7699,7 +7699,37 @@ <h3><dfn>RTCDtlsTransport</dfn> Interface</h3>
           at <var>transport</var>.</p>
         </li>
       </ol>
-
+      <p>When the underlying DTLS transport experiences an error, such as
+        a certificate validation failure, or a fatal alert (see RFC 5246
+        section 7.2), the user agent MUST queue a task that runs the following
+        steps:
+      </p>
```

Comment:
nit: Here and elsewhere - There shouldn't be an extra indent here and &lt;/p&gt; should be on the line above